Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -48,10 +48,12 @@ function Initialize-DevelopmentEnvironment {
         Builds the ODS/API solution and deploys the necessary databases in order to setup a complete development environment.
     .parameter InstallType
         The type of deployment to install: 'Sandbox', 'SharedInstance', 'YearSpecific', or 'DistrictSpecific'.
+        InstallType variable is an string variable ,so you can use single quotes or double quotes both are supported format ,when you set this one value in octopus.
```

Comment:
quotes aren't required here even in Octopus so I think we can remove this

Reply:
I removed it